doc/articles/getting-started/deployments.md

```diff
@@ -0,0 +1,167 @@
+
+-   `application/wasm`
+-   `application/octet-stream`
+-   `application/octet-stream`
+-   `application/font-woff`
+-   `application/font-woff`
+
+## Nginx
+
+Below contains a extremly performance tuned version of *nginx* fitting WASM deployments. The MIME types have been extended on top of the base *nginx* MIME types with WASM support.
+
+```nginx
+user nginx;
+
+pid /var/run/nginx.pid;
+
+##################################################################################
+# nginx.conf Performance Tuning: https://github.com/denji/nginx-tuning
+##################################################################################
+
+# you must set worker processes based on your CPU cores, nginx does not benefit from setting more than that
+worker_processes auto; #some last versions calculate it automatically
+
+# number of file descriptors used for nginx
+# the limit for the maximum FDs on the server is usually set by the OS.
+# if you don't set FD's then OS settings will be used which is by default 2000
+worker_rlimit_nofile 100000;
+
+# only log critical errors
+error_log /var/log/nginx/error.log crit;
+
+# provides the configuration file context in which the directives that affect connection processing are specified.
+events {
+  # determines how much clients will be served per worker
+  # max clients = worker_connections * worker_processes
+  # max clients is also limited by the number of socket connections available on the system (~64k)
+  worker_connections 4000;
+
+  # optmized to serve many clients with each thread, essential for linux -- for testing environment
+  use epoll;
+
+  # accept as many connections as possible, may flood worker connections if set too low -- for testing environment
+  multi_accept on;
+}
+
+http {
+  include /etc/nginx/mime.types;
+  default_type application/octet-stream;
+
+  types {
+    application/wasm wasm;
+    application/octet-stream clr;
+    application/octet-stream pdb;
+    application/font-woff woff;
+    application/font-woff woff2;
+  }
+
+  log_format main '$remote_addr - $remote_user [$time_local] "$request" '
+  '$status $body_bytes_sent "$http_referer" '
+  '"$http_user_agent" "$http_x_forwarded_for"';
+
+  # cache informations about FDs, frequently accessed files
+  # can boost performance, but you need to test those values
+  open_file_cache max=200000 inactive=20s;
+  open_file_cache_valid 30s;
+  open_file_cache_min_uses 2;
+  open_file_cache_errors on;
+
+  # to boost I/O on HDD we can disable access logs
+  access_log off;
+
+  # copies data between one FD and other from within the kernel
+  # faster then read() + write()
+  sendfile on;
+
+  # send headers in one peace, its better then sending them one by one
+  tcp_nopush on;
+
+  # don't buffer data sent, good for small data bursts in real time
+  tcp_nodelay on;
+
+  # reduce the data that needs to be sent over network -- for testing environment
```

Comment:
Why is this test environment only ? Wasm apps benefit a lot from compression, and even better if brotli can be enabed.

Reply:
Added Brotli & removed test environment comment.